spec.go

```diff
@@ -53,6 +57,8 @@ type Process struct {
 	// Cwd is the current working directory for the process and must be
 	// relative to the container's root.
 	Cwd string `json:"cwd"`
+	// ExtraFiles specifies open file descriptors that should be passed to the process.
+	ExtraFiles []os.File `json:"extraFiles"`
```

Comment:
On Fri, Aug 21, 2015 at 11:16:46AM -0700, Michael Crosby wrote:
> > +	// ExtraFiles specifies open file descriptors that should be passed to the process.
> > +	ExtraFiles []os.File `json:"extraFiles"`
> 
> i think having runtime support for `LISTEN_FDS` and doing
> passthrough would be better than adding this runtime information in
> the spec that is to be serialized.

You're still serializing those file descriptors into the environment
variable, no?  With #88 and the LISTEN_FDS suggestion, that would mean
the runtime would have to check:

* The static config.json
* The more dynamic runtime.json
* The LISTEN_FDS environment variable

to figure out how to create the container and launch the application.
Personally, I see no problem with mutating a single config to launch
the container (so the runtime only has to look in one place) [1,2].
But if we do end up getting a second place to put any configuration
considered “too host-specific for config.json”, then I don't see a
need to go further than a runtime.json.

[1]: https://github.com/opencontainers/specs/pull/88#issuecomment-126516625
[2]: https://github.com/opencontainers/runc/issues/200#issuecomment-131404037